docs/advanced-features/compiler.md

```diff
@@ -241,18 +242,18 @@ Only `importMap` in `@emotion/babel-plugin` is not supported for now.
 
 ### Minification
 
-You can opt-in to using the Next.js compiler for minification. This is 7x faster than Terser.
+Next.js' swc compiler is used for minification by default since v13. This is 7x faster than Terser.
```

Comment:
I think "swc compiler" is redundant since swc stands for "speedy web compiler"

Also, this should be added to `upgrading.md` too, right?